Dockerfile

```diff
@@ -9,10 +9,8 @@ FROM base AS build
 # Build envs
 ARG BUILD_DATE
 ENV BUILD_DATE=$BUILD_DATE
-ARG ENV_EXAMPLE
-ENV ENV_EXAMPLE=$ENV_EXAMPLE
-ARG NEXT_PUBLIC_ENV_EXAMPLE
-ENV NEXT_PUBLIC_ENV_EXAMPLE=$NEXT_PUBLIC_ENV_EXAMPLE
+ARG AEM_GRAPHQL_ENDPOINT
+ENV AEM_GRAPHQL_ENDPOINT=$AEM_GRAPHQL_ENDPOINT
```

Comment:
If the env variable is added at runtime, pretty sure we determined it's not ALSO required a build...

Reply:
ahh good point, so this env var might be an exception to that "rule"

Reply:
It's not a rule parse, different environment variables have different purposes. As long as the variable isn't prefixed with next_public they should be relatively safe